tools/cythonize.py

```diff
@@ -52,9 +52,9 @@
 # Rules
 #
 def process_pyx(fromfile, tofile):
-    flags = ['-3', '--fast-fail']
+    flags = ['--fast-fail']
```

Comment:
Why do we drop `-3` here?

Reply:
```
charris@fc [~]$ python3 -3
Unknown option: -3
```

Reply:
`-3` just optimizes code by removing py2 code paths. so shouldn't give test failures either way